app/actions/websocket/index.ts

```diff
@@ -104,6 +104,8 @@ async function doReconnect(serverUrl: string, groupLabel?: BaseRequestGroupLabel
 
     dataRetentionCleanup(serverUrl);
 
+    expiredBoRPostCleanup(serverUrl);
```

Comment:
Do we have a test where we can assert this is being called?

Reply:
Done, added now to assert that this was called.

Reply:
Also, do we want to await for the `dataRententionCleanup` to finish or call it from `dataRetentionCleanup`? I wonder if we may have a conflict trying to delete the same posts or something like that, or that scenario can never happen?